docs/docs/install/docker-compose.mdx

```diff
@@ -2,6 +2,9 @@
 sidebar_position: 30
 ---
 
+import CodeBlock from '@theme/CodeBlock';
+import ExampleEnv from '!!raw-loader!../../../docker/example.env';
```

Comment:
This just inlines the actual file contents directly in the docs.

Reply:
That's freaking awesome

Reply:
That's freaking wild syntax!